xlators/mgmt/glusterd/src/glusterd-tierd-svc-helper.c

```diff
@@ -107,7 +107,6 @@ glusterd_svc_check_tier_volfile_identical(char *svc_name,
 
     this = THIS;
 
-    GF_VALIDATE_OR_GOTO(THIS->name, this, out);
```

Comment:
The whole function - glusterd_svc_check_tier_volfile_identical(), seems to be dead code...

Reply:
I think I missed this...